tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1277,16 +1356,14 @@ void test() {
 
     libfmt_formatter_test_zero_flag<char>();
     libfmt_formatter_test_zero_flag<wchar_t>();
+
+    libfmt_formatter_test_runtime_width<char>();
+    libfmt_formatter_test_runtime_width<wchar_t>();
+
+    libfmt_formatter_test_runtime_precision<char>();
+    libfmt_formatter_test_runtime_precision<wchar_t>();
 }
 
 int main() {
-    try {
-        test();
-    } catch (const format_error& e) {
-        printf("format_error: %s\n", e.what());
-        assert(false);
-    } catch (const exception& e) {
-        printf("exception: %s\n", e.what());
-        assert(false);
-    }
+    test();

```

Comment:
Why is this try/catch machinery being dropped? I added this when investigating test failures (otherwise unhandled exceptions result in an error code of 3 which is not informative).

Reply:
unfortunately I removed them for the same reason! they make investigating the errors with a debugger really hard, since you'll either have to first chance on format_error (there are a lot of expected format_errors thrown in the tests) or you'll break on the abort and the originating callstack is gone.

Reply:
Ok, makes sense.